src/test/typescript-service-helpers.ts

```diff
@@ -1577,12 +1577,15 @@ export function describeTypeScriptService(TypeScriptServiceConstructor = TypeScr
 
 		beforeEach(<any> initializeTypeScriptService(TypeScriptServiceConstructor, new Map([
 			['file:///keywords-in-path/class/constructor/a.ts', 'export function a() {}'],
-			['file:///special-characters-in-path/%40foo/b.ts', 'export function b() {}']
+			['file:///special-characters-in-path/%40foo/b.ts', 'export function b() {}'],
+			['file:///windows/app/master.ts', '/// <reference path="..\\lib\\master.ts" />\nc();'],
+			['file:///windows/lib/master.ts', '/// <reference path="..\\lib\\slave.ts" />'],
+			['file:///windows/lib/slave.ts', 'function c() {}']
 		])));
 
 		afterEach(<any> shutdownTypeScriptService);
 
-		describe('getHovers()', <any> function (this: TestContext) {
+		describe('LSP server', <any> function (this: TestContext) {
```

Comment:
Well, with the addition of new test scenario `getHover()` is no longer reflects properly what we are testing. New test case uses `getDefinition` because `getHover()` does not provide enough proof of correctness in the triple-slash references case.
Also, frankly I don't care what we are using to test server behavior when dealing with not-common file paths, my goal is just to test that we can handle them